src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +13,6 @@ public class Users {
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users.stream().filter(user -> user.matchName(name)).findAny().orElseGet(() -> DEFAULT_USER);
```

Comment:
스트림 메서드마다 줄바꿈을 해주면 가독성이 올라갑니다!
```suggestion
        return users.stream()
                .filter(user -> user.matchName(name))
                .findAny()
                .orElseGet(() -> DEFAULT_USER);
```